src/storage/disk_manager.c

```diff
@@ -599,8 +598,9 @@ disk_format (THREAD_ENTRY * thread_p, const char *dbname, VOLID volid, DBDEF_VOL
   vhdr->nsect_max = ext_info->nsect_max;
   vhdr->db_charset = lang_charset ();
   vhdr->hint_allocsect = NULL_SECTID;
-  vhdr->dummy1 = vhdr->dummy2 = 0;	/* alignment. You may use it. */
-  vhdr->reserved0 = vhdr->reserved1 = vhdr->reserved2 = vhdr->reserved3 = 0;	/* for future extension */
+  vhdr->vol_creation_time = time (NULL);
+  vhdr->dummy1 = vhdr->dummy2 = vhdr->dummy3 = 0;	/* alignment. You may use it. */
+  vhdr->reserved0 = 0;		/* for future extension */
```

Comment:
```suggestion
  vhdr->dummy1 = vhdr->dummy2 = vhdr->dummy3 = 0;
  vhdr->reserved0 = 0;
```
구조체에 이미 설명되어 있는 내용에 대해 주석으로 남길 필요는 없어보입니다. 기존에 있던 주석을 삭제해 주시길 바랍니다.

Reply:
삭제하였습니다.